packages/backend/src/peer/model.ts

```diff
@@ -46,6 +46,8 @@ export class Peer
 
   public staticIlpAddress!: string
 
+  public name?: string
```

Comment:
The column needs to be added to `peers` table.
You can add it to the migration file:
https://github.com/interledger/rafiki/blob/4db6f8e8ee6e4812af452af379855e4c3ff8d2f6/packages/backend/migrations/20211026133430_create_peers_table.js#L2
The create peer insert should also include `name`
https://github.com/interledger/rafiki/blob/4db6f8e8ee6e4812af452af379855e4c3ff8d2f6/packages/backend/src/peer/service.ts#L103-L109